activelearning.py

```diff
@@ -1,349 +1,63 @@
+"""
+Active learning with GFlowNets
```

Comment:
This looks good to me for now, except that I would not make it a class. But it's great for now!